cmd/cli/vocdonicli.go

```diff
@@ -114,7 +114,11 @@ func (v *vocdoniCLI) setHost(host string) error {
 	if err != nil {
 		return err
 	}
-	v.api.SetHostAddr(u)
+	err = v.api.SetHostAddr(u)
```

Comment:
Hey Maria.

It is generally recommended to use the short variable declaration operator := when initializing a new variable in the same line that it is used. This creates the variable in the closest scope possible, which can help make the code more readable and reduce the chance of bugs. Using := also helps to avoid reusing variables that might have been defined earlier in the code and might have different values.

Additionally, it is considered a good practice to avoid reusing error variables, as it can lead to confusion and make it harder to understand the code. Instead, it is recommended to create a new variable for each error that is returned, and to name the variable something that clearly indicates its purpose.

To sum it up, it is suggested to use `if err := v.api.SetHostAddr(u)` instead of `err = v.api.SetHostAddr(u)` to keep the code more readable, and avoid potential bugs and confusion.

Please apply this principle to the following modified lines.

Reply:
Hey Pau! 

I get it, thank you! I will make the changes.